tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -109,6 +106,9 @@ export class WebGPUBackend extends KernelBackend {
 
   constructor(device: GPUDevice, glslang: Glslang, supportTimeQuery = false) {
     super();
+    if (!env().getBool('HAS_WEBGPU')) {
+      throw new Error('WebGPU is not supported on this device');
+    }
```

Comment:
Since `env().getBool('HAS_WEBGPU')` has been checked in `index.ts`, does it mean you don't need the repeated checking here?